reports/summary-2021.md

```diff
@@ -0,0 +1,1191 @@
+  - [2802 -Identify Windows pods at API admission level authoritatively](https://git.k8s.io/enhancements/keps/sig-windows/2802-identify-windows-pods-apiserver-admission) - v1.23
+- Pre-alpha (Targeting 1.24)
+  - [2578 - Windows Operational Readiness](https://git.k8s.io/enhancements/keps/sig-windows/2578-windows-conformance/)
+
+[`hostProcess`]: https://git.k8s.io/enhancements/keps/sig-windows/1981-windows-privileged-container-support/
+[sig-windows-dev-tools]: https://github.com/kubernetes-sigs/sig-windows-dev-tools
+
+
+#### [WG API Expression](https://git.k8s.io/community/wg-api-expression/annual-report-2021.md#current-initiatives)
+
+- Server-side Apply went GA
+- Started new initiatives around OpenAPI v3
+- Enum for built-in types in OpenAPI
+- Server-side field validation
+
+
+#### [WG Data Protection](https://git.k8s.io/community/wg-data-protection/annual-report-2021.md#current-initiatives)
+
+The Data Protection WG identified the missing building blocks for supporting
+data protection in Kubernetes and published in their [whitepaper]. Features
+such as Volume Backups, Change Block Tracking, Volume Populator, Volume Group
+Group Snapshot, and Backup Repositories are owned by SIG Storage. Features such
+as Quiesce and Unquiesce Hooks are owned by SIG Node, with SIG Storage and SIG
+Apps participating. Features such as Application Snapshots and Backups are
+owned by SIG Apps, with SIG Storage participating. We will continue to work on
+them until all the missing pieces are available in Kubernetes.
+
+The following items have been under development and have not yet been captured
+in a KEP:
+- [Change Block Tracking (CBT) API design](https://docs.google.com/document/d/15tLCV3csvjHbKb16DVk-mfUmFry_Rlwo-2uG6KNGsfw/edit#heading=h.1olwavha9frv)
+- [Volume Replication](https://docs.google.com/document/d/15tLCV3csvjHbKb16DVk-mfUmFry_Rlwo-2uG6KNGsfw/edit#heading=h.pah3yke9ddug)
+- [Data Protection for Managed Services Presentation](https://docs.google.com/presentation/d/1IM6d0w3CDdHv1dLaFNXEcxy5fuDTr9LERAdMVkZiK9s/edit#slide=id.p)
+- [Snapshot policy (immutable snapshot](https://docs.google.com/document/d/15tLCV3csvjHbKb16DVk-mfUmFry_Rlwo-2uG6KNGsfw/edit#heading=h.gb8m7t8jro1v)
+- [Volume Snapshot GA phases](https://docs.google.com/document/d/15tLCV3csvjHbKb16DVk-mfUmFry_Rlwo-2uG6KNGsfw/edit#heading=h.w8v8tpkuw8ac)
+- [Kubernetes Data Protection with Velero](https://docs.google.com/document/d/15tLCV3csvjHbKb16DVk-mfUmFry_Rlwo-2uG6KNGsfw/edit#heading=h.iekhl8nl58lo)
+
+
+
+[whitepaper]: https://github.com/kubernetes/community/blob/master/wg-data-protection/data-protection-workflows-white-paper.md#what-are-the-missing-building-blocks-in-kubernetes
+
+
+#### [WG IoT/Edge](https://git.k8s.io/community/wg-iot-edge/annual-report-2021.md#current-initiatives)
+
+The IoT/Edge Working Ground is moving to the CNCF ecosystem. Home to be announced soon.
```

Comment:
Does this need more detail? Maybe skip the “Home to be announced soon” part here?